integrations/extensions/starter-kits/coveo/README.md

```diff
@@ -39,73 +41,74 @@ If you want to add this starter kit to an _existing_ assistant, you cannot use t
 - Download the OpenAPI specification in this starter kit.
 - Use the OpenAPI specification to [build a custom extension](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension).
 - [Add the extension to your assistant](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-add-custom-extension) using the API key you obtained in the pre-requisites step above.
-- Go to `Variables > Created by you` and add `query_text`, `result_items` `link0`, `link1`, `link2`, `title0`, `title1`, `title2`, `snippet0`,`snippet1`, and `snippet2`.
+- Go to `Variables > Created by you` and add `query_text`, `search_results`, `search_result`, `link`, `title`, and `snippet`.
 - Create a new action and put "Search" in "What does your customer say to start this interaction?".  Add step 1:
```

Comment:
At this time, you should probably also create the "Show search results" and "Show search result" actions.  You fill in the details for those two later on, but they need to be created now because your instructions wind up _using_ them before you fill in all the details for them, which I think seems like a reasonable organization.